velox/vector/tests/VectorTest.cpp

```diff
@@ -18,6 +18,7 @@
 #include <gtest/gtest.h>
 #include <cstdint>
 #include <functional>
+#include <iostream>
```

Comment:
Would you remove this include and rebase?

Reply:
@mbasmanova Done.

Reply:
Thanks.